etc/dbus-serialbattery/config.default.ini

```diff
@@ -202,8 +202,17 @@ CUSTOM_BATTERY_NAMES =
 
 ; Auto reset SoC
 ; If on, then SoC is reset to 100%, if the value switches from absorption to float voltage
-; Currently only working for Daly BMS
+; Currently only working for Daly and JK BMS
 AUTO_RESET_SOC = True
+; --------- JK BMS specific settings for SOC reset ---------
+; WARNING! Beta code only tested with HW 11!
+; WARNING! This will always set your JK BMS OVP and OVPR values to this settings!
+JK_BMS_AUTO_RESET_SOC = False
+JK_BMS_OVPR_DEFAULT = 3.450
+JK_BMS_OVP_DEFAULT = 3.550
+JK_BMS_OVPR_TRIGGER_RESET = 3.300
+JK_BMS_OVP_TRIGGER_RESET = 3.400
+JK_BMS_WAIT_UNTIL_NEXT_SOC_RESET = 3600
```

Comment:
You don't need all this complicated settings. This is only generating a lot of questions for the users.

The way to go should be:
1. Read OVP and OVPR from BMS and save it to variables
2. Disable the alert
3. Set OVP to `MAX_CELL_VOLTAGE` and OVPR to `MAX_CELL_VOLTAGE` - `0.25` 
4. Check if SoC is reseted to 100%
5. Set OVP and OVPR to the original values
6. Enable alert again

If you let the user decide `JK_BMS_WAIT_UNTIL_NEXT_SOC_RESET` then one of the most questions will be: What is the best value to set? To avoid this please set a fixed value as a constant in the script. I would recommend that SoC is set only once a day.